pkgs/development/tools/rep/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, asciidoc-full }:
+
+stdenv.mkDerivation rec {
+  pname = "rep";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "eraserhd";
+    repo = "rep";
+    rev = "f71dfc3c99448827dd209f66ac20456a5d988506";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
This was because of a nix-specific Makefile tweak committed after release, but I made a point release to clear it up.